src/staging/validation.py

```diff
@@ -50,9 +49,7 @@ def get_masterlist(postcode_masterlist) -> pd.Series:
 
```

Comment:
Line 21: incomplete sentence
Could add to doc string : "masterlist contains - all post codes in the UK" if that's correct
Line 39: is the doc string correct? Maybe move get_masterlist to above check_pcs_real so they can be read together more easily